pulpcore.fc

```diff
@@ -8,15 +8,9 @@
 /etc/pulp/private_key.pem			gen_context(system_u:object_r:pulpcore_etc_t,s0)
 /etc/pulp/public_key.pem			gen_context(system_u:object_r:pulpcore_etc_t,s0)
 
-/usr/bin/rq		--	gen_context(system_u:object_r:pulpcore_exec_t,s0)
-/usr/bin/gunicorn	--	gen_context(system_u:object_r:pulpcore_server_exec_t,s0)
-
 /usr/libexec/pulpcore/.*	--	gen_context(system_u:object_r:pulpcore_exec_t,s0)
 /usr/libexec/pulpcore/gunicorn	--	gen_context(system_u:object_r:pulpcore_server_exec_t,s0)
 
-/usr/local/lib/pulp/bin/gunicorn	--	gen_context(system_u:object_r:pulpcore_server_exec_t,s0)
-/usr/local/lib/pulp/bin/rq		--	gen_context(system_u:object_r:pulpcore_exec_t,s0)
-
```

Comment:
Do not remove (yet.) We use these for when Pulp is installed by pulp_installer from pip mode rather than from RPM packages.

I have yet to determine how these projects (pulp_installer / pulpcore-selinux) will handle the path being changed. However, for now, if users do not change the path when they install in pip mode, it works.

This is our epic for this work:
https://pulp.plan.io/issues/7043

Reply:
Should I split this up into two PRs so we can merge one and keep the other blocked on that issue?

Reply:
@ekohl I'll submit that PR when it's ready. I have yet to figure out how I'll handle it even. Will I template this file? Cancatenate it with a templated file? It's TBD, and depends partially on implementation (like what ansible modules exist, and can be made declarative, and the fact that the git repo tracks the file too.)